tests/components/jvc_projector/test_sensor.py

```diff
@@ -0,0 +1,22 @@
+"""Tests for the JVC Projector binary sensor device."""
+
+from unittest.mock import MagicMock
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from tests.common import MockConfigEntry
+
+ENTITY_ID = "sensor.jvc_projector_power_status"
+
+
+async def test_entity_state(
+    hass: HomeAssistant,
+    entity_registry: er.EntityRegistry,
+    mock_device: MagicMock,
+    mock_integration: MockConfigEntry,
+) -> None:
+    """Tests entity state is registered."""
+    entity = hass.states.get(ENTITY_ID)
+    assert entity
+    assert entity_registry.async_get(entity.entity_id)
```

Comment:
What are you even testing?

Reply:
I'm testing that the entity is created correctly. As it does not have any direct logic but relies on the coordinator get_status with the communication with the projector I do not need to test anything else.

Reply:
Yes you do, you're adding two entities, at least test the state